addons/cloud-games/addon.json

```diff
@@ -11,8 +11,25 @@
   ],
   "popup": {
     "icon": "../../images/icons/cloud.svg",
-    "name": "Games"
+    "name": "Games",
+    "script": "popup.js",
+    "html": "popup.html"
```

Comment:
Make script, to an array?

To be consistent, these values should be relative to the addon's folder? I'm trying to say that you should remove the popups folder and put it's files in the addons's folders.

Reply:
Not gonna happen for security reasons.

Reply:
I can make scripts an array. Not sure if it's that useful tho

Reply:
What reasons?

Reply:
Any websites could embed popups, which is not good for obvious reasons.

Reply:
wut... im asking you to remove the popups folder and move it to addons folder.

Reply:
And that's the consequence. Any websites could embed popups if you do that (at least on Chrome with MV2), which is not good for obvious reasons.

Reply:
A single script is fine. "import" can always be used if needed.